File: game-server/src/main/java/work/fking/pangya/game/packet/outbound/MascotRosterPacket.java
package work.fking.pangya.game.packet.outbound;

import io.netty.buffer.ByteBuf;
import work.fking.pangya.networking.protocol.OutboundPacket;

public class MascotRosterPacket  implements OutboundPacket {

    private static final int ID = 0xe1;

    @Override
    public void encode(ByteBuf target) {
        target.writeShortLE(ID);

        target.writeShortLE(0); // count
    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/outbound/MascotRosterPacket.kt
package work.fking.pangya.game.packet.outbound

import io.netty.buffer.ByteBuf
import work.fking.pangya.networking.protocol.OutboundPacket

class MascotRosterPacket : OutboundPacket {

    override fun encode(target: ByteBuf) {
        target.writeShortLE(0xe1)
        target.writeShortLE(0) // count
    }
}
